Admin CLI :: Docker build fails

```
[INFO] DOCKER> --------------------
13:09:57   [INFO] DOCKER>   28 |         name="Spline Admin Tool"
13:09:57   [INFO] DOCKER>   29 |
13:09:57   [INFO] DOCKER>   30 | >>> RUN apk add --no-cache tini=0.19.0-r3 curl=8.14.1-r1
13:09:57   [INFO] DOCKER>   31 |
13:09:57   [INFO] DOCKER>   32 |     USER 1001
13:09:57   [INFO] DOCKER> --------------------
13:09:57   [INFO] DOCKER> ERROR: failed to solve: process "/bin/sh -c apk add --no-cache tini=0.19.0-r3 curl=8.14.1-r1" did not complete successfully: exit code: 1
13:09:57   [ERROR] DOCKER> Error status (1) when building
13:09:57 ```

--- a/admin/Dockerfile
+++ b/admin/Dockerfile
@@ -27,7 +27,7 @@ LABEL \
     license="Apache License, version 2.0" \
     name="Spline Admin Tool"
 
-RUN apk add --no-cache tini=0.19.0-r3 curl=8.14.1-r1
+RUN apk add --no-cache 'tini~=0.19' 'curl~=8.14'
 
 USER 1001
 

